services/gateway/apis/http-version3/swagger/definitions/transactions.json

```
@@ -10,60 +10,162 @@
                 "$ref": "#/definitions/TransactionsStatistics"
             },
             "meta": {
-                "$ref": "#/definitions/Pagination"
+                "$ref": "#/definitions/TransactionStatisticsMeta"
+            }
+        }
+    },
+    "TransactionStatisticsMeta": {
+        "type": "object",
+        "required": [
+            "count",
+            "offset",
+            "total"
+        ],
+        "properties": {
+            "count": {
+                "type": "integer",
+                "example": 10,
+                "minimum": 0,
+                "description": "Number of items returned in the request."
+            },
+            "offset": {
+                "type": "integer",
+                "example": 10,
+                "minimum": 0,
+                "description": "Number of items skipped in the response."
+            },
+            "total": {
+                "type": "integer",
+                "example": 100,
+                "minimum": 0,
+                "description": "Number of items returned in the response."
+            },
+            "date": {
+                "type": "object",
+                "properties": {
+                    "dateFormat": {
+                        "type": "string",
+                        "example": "YYYY-MM"
+                    },
+                    "dateFrom": {
+                        "type": "string",
+                        "example": "2021-12"
+                    },
+                    "dateTo": {
+                        "type": "string",
+                        "example": "2022-09"
+                    }
+                }
+            },
```

Also, please consider adding descriptions to each of the properties listed below.
```suggestion
            "duration": {
                "type": "object",
                "properties": {
                    "format": {
                        "type": "string",
                        "example": "YYYY-MM"
                    },
                    "from": {
                        "type": "string",
                        "example": "2021-12"
                    },
                    "to": {
                        "type": "string",
                        "example": "2022-09"
                    }
                }
            },
```